posthog/models/team.py

```diff
@@ -65,14 +65,22 @@ def create(self, *args, **kwargs) -> "Team":
             raise ValueError("Creating organization-less projects is prohibited")
         return super().create(*args, **kwargs)
 
-    def get_team_from_token(self, token: Optional[str]) -> Optional["Team"]:
+    def get_team_from_token(self, token: Optional[str], only: Tuple) -> Optional["Team"]:
         if not token:
             return None
         try:
-            return Team.objects.get(api_token=token)
-        except Team.DoesNotExist:
+            return Team.objects.filter(api_token=token).only(*only)[0]
+        except IndexError:
             return None
 
+    def get_team_cached(self, pk: int) -> "Team":
```

Comment:
Q: Why is this quoted vs using the type?

Reply:
Team isn't yet defined

Reply:
Woah that is crazy. Didn't know that was an option.